app/pb_kits/playbook/pb_label_value/_label_value.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.label %>
-  <%= object.value %>
+    class: object.classname) do %>
+  <%= pb_rails("caption", props: { text: "#{object.label}" }) if object.label? %>
+  <%= pb_rails("body", props: { text: "#{object.value}" }) %>
```

Comment:
No string interpolation is needed because `label` and `value` are already strings.